### PR TITLE
Add missing RGB_MODE_TWINKLE / RGB_M_TW keycodes

### PR DESCRIPTION
**Pull request by spidey3**
## Description

When the Twinkle effect was implemented, keycodes were not created to select it.
This adds them.

## Types of Changes

- [x] Core
- [x] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by fauxpark** [comment]
Just a note, this will break VIA. At least the Space Cadet keycodes, and maybe EEPROM reset, though I can't find it in the list.

**Comment by spidey3** [comment]
> Just a note, this will break VIA. At least the Space Cadet keycodes, and maybe EEPROM reset, though I can't find it in the list.

Ooh, ouch.
Should I re-point it at develop?

**Review by drashna — APPROVED** [review]

**Comment by tzarc** [comment]
Sorry about that, GitHub decided to delete the `develop` branch from upstream when we merged it, despite being told otherwise. Reopened.

**Comment by spidey3** [comment]
Rebased, taken off hold.

**Review by drashna — APPROVED** [review]

File: quantum/quantum_keycodes.h [hunk 1]
```diff
@@ -511,6 +511,9 @@ enum quantum_keycodes {
     ONESHOT_DISABLE,
     ONESHOT_TOGGLE,
 
+    // Miscellaneous recent additions
```
Comment (fauxpark) [inline quantum/quantum_keycodes.h:514]:
```suggestion
    // RGB underglow/matrix (continued)
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Pull request merged**